### PR TITLE
Update language recommendations

### PR DESCRIPTION
**Pull request by samoylenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,117 +156,154 @@ controls from scratch. Custom security controls implementation and not using
...
-stack. Combined with a widespread community of helpful and friendly developers,
-Java is a great first language to learn.
+Widely used by the enterprise, it's not going away anytime soon. Unlike
+proprietary technology stacks like Microsoft's (.Net/C#) or Apple's, it's free
+to use and learn. Combined with a widespread community of helpful and friendly
+developers, Java is a great first language to learn.
 
-If you don't like Java, our other recommendation would be **Go**, but
-it's not currently very popular in the world of enterprise.
+If you don't like Java, our other recommendation would be **Go**, but it's not
```
Comment (wasciutto) [inline README.md]: A brief explanation of why Go is recommended would be helpful here

File: README.md [hunk 2]
```diff
@@ -151,117 +156,154 @@ controls from scratch. Custom security controls implementation and not using
...
+        Don't be alarmed by the book's size. You only need to read the first
+        3 sections: "*Getting started*", "*Git basics*", and "*Git branching*" to
+        start. It will take around an hour of your time.
+
+    2.  Git also has very cool tutorials: <https://try.github.io>
+
+3.  We also recommend you to start practicing
+    [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development)
+    \- this is one of the most important concepts that you will need to be
+    familiar with in your job as an Application Security professional. By the
```
Comment (wasciutto) [inline README.md]: Why?

Comment (samoylenko) [reply README.md]: Added in the next commit

File: README.md (hunk 1 above)
Comment (samoylenko) [reply README.md]: Added in the next commit. Just the same quality of 'why' as for Java: popularity and feedback we heard. What I won't mention there: basically, there are not so many popular languages to recommend: C, C++ -- filtered out, not popular. Java and C#/VB .NET - mentioned already,
Pascal - joke, Kotlin / Groovy - built on top of JVM, so in this text scope its better to recommend starting with Java, Go and Rust remain =) But rust is far less seen in Enterprise. Hence. Go :)

**Pull request merged**